torch/quantization/_numeric_suite.py

```diff
@@ -211,12 +227,18 @@ def __init__(self, q_module, float_module, Logger):
         self.dequant = nnq.DeQuantize()
         self.logger = Logger()
 
-    def forward(self, x):
-        output = self.orig_module(x)
-        if x.is_quantized:
-            x = x.dequantize()
-        shadow_output = self.shadow_module(x)
-        self.logger(output, shadow_output)
+    def forward(self, *x):
+        if len(x) > 1:
```

Comment:
Shadow module should pass all arguments to orig and shadow_module. The logger should also take in the entire outputs. Internally the logger could choose to log output[0], at the forward here, lets keep it general.